ci-operator/config/openshift/installer/openshift-installer-master__terraform.yaml

```diff
@@ -0,0 +1,96 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.20-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |
```

Comment:
The `dockerfile_literal` is good to be able to trigger rehearsals and CI config changes in a single PR, but at the end we want this dockerfile to live in the installer, repo, right?

Reply:
Right, this is convenient right now for experimenting without impacting too many repos. Once we'll be happy of the final dockerfile shape, we could refactor and extract it into its own file in the installer repo, and remove the literal here.